presto-hive/pom.xml

```diff
@@ -171,6 +171,12 @@
         </dependency>
 
         <dependency>
+            <groupId>com.facebook.presto</groupId>
```

Comment:
This is problematic because presto-client is not meant to be visible from server to plugins across classloader boundaries. Even if it works today, it may break once we fix classloader isolation.